docs/reference/api/usb/USBMIDI.md

```diff
@@ -0,0 +1,31 @@
+## USBMIDI
+
+<span class="images">![](https://os.mbed.com/docs/v5.9/feature-hal-spec-usb-device-doxy/class_u_s_b_m_i_d_i.png)<span>USBMIDI class hierarchy</span></span>
+
+You can use the USBMIDI interface to send and receive MIDI messages over USB using the standard USB-MIDI protocol.
+
+Examples of tasks you can perform using this library include sending MIDI messages to a computer (such as to record in a sequencer, or trigger a software synthesiser) and receiving messages from a computer (such as actuating things based on MIDI events).
```

Comment:
What are some best practices for using USBMIDI? What are some concepts you need to understand to use it?